_haikus/CStamp.md

```diff
@@ -0,0 +1,3 @@
+clouds are friends not food
```

Comment:
Hi there, please take a look at the [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) so that you format this haiku correctly! It needs to follow the proper template in order for the page to be interpreted correctly.